src/app/pages/search/search.module.js

```diff
@@ -7,6 +7,7 @@ import ChplRealWorldTestingSearchWrapper from './real-world-testing/real-world-t
 import ChplListingsSearchWrapper from './listings/listings-wrapper';
 import ChplSedSearchWrapper from './sed/sed-wrapper';
 import ChplSvapSearchWrapper from './svap/svap-wrapper';
+import ChplDecisionSupportInterventionsSearchWrapper from './decision-support-interventions/decision-support-interventions-wrapper';
```

Comment:
Almost all of our imports are sorted alphabetically. This ought to be after the ChplDecertified import. The bridges that are created below are also sorted in the same way as the imports